### PR TITLE
Document the scrape_duration_seconds timeseries

### PR DESCRIPTION
**Pull request by stapelberg**

### CODE REVIEW

File: content/docs/concepts/jobs_instances.md [hunk 1]
```diff
@@ -30,7 +30,12 @@ depends on the `honor_labels` configuration option. See the
 [scrape configuration documentation](/docs/operating/configuration/#scrape-configurations-scrape_config)
 for more information.
 
-For each instance scrape, Prometheus stores a sample of the form
-`up{job="<job-name>", instance="<instance-id>"}` with a value of `1` if the
-instance was scraped successfully or a value of `0` if the scrape failed. This
-time series is useful for instance availability monitoring.
+For each instance scrape, Prometheus stores a simple in the following
```
Comment (brian-brazil) [inline content/docs/concepts/jobs_instances.md]: sample

File: content/docs/concepts/jobs_instances.md [hunk 2]
```diff
@@ -30,7 +30,12 @@ depends on the `honor_labels` configuration option. See the
...
+time series:
+
+* `up{job="<job-name>", instance="<instance-id>"}`: `1` if the instance is
+   healthy, i.e. reachable, or `0` if the scrape failed.
+* `scrape_duration_seconds{job="<job-name>", instance="<instance-id>"}`:
+  duration of the last scrape
```
Comment (brian-brazil) [inline content/docs/concepts/jobs_instances.md]: Indentation is off here, and missing a full stop

File: content/docs/concepts/jobs_instances.md [hunk 3]
```diff
@@ -30,7 +30,12 @@ depends on the `honor_labels` configuration option. See the
...
+
+These time series are useful for instance availability monitoring.
```
Comment (brian-brazil) [inline content/docs/concepts/jobs_instances.md]: This is only true of `up`.

**Comment by stapelberg** [comment]
All done. PTAL.

File: content/docs/concepts/jobs_instances.md [hunk 4]
```diff
@@ -30,7 +30,12 @@ depends on the `honor_labels` configuration option. See the
...
-`up{job="<job-name>", instance="<instance-id>"}` with a value of `1` if the
-instance was scraped successfully or a value of `0` if the scrape failed. This
-time series is useful for instance availability monitoring.
+For each instance scrape, Prometheus stores a sample in the following
+time series:
+
+* `up{job="<job-name>", instance="<instance-id>"}`: `1` if the instance is
+   healthy, i.e. reachable, or `0` if the scrape failed.
+* `scrape_duration_seconds{job="<job-name>", instance="<instance-id>"}`:
+   duration of the last scrape.
```
Comment (brian-brazil) [inline content/docs/concepts/jobs_instances.md]: s/last //

**Comment by stapelberg** [comment]
Done.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**